git_hub_links.rb

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require './slack_bot'
+##
+# This script uses the GitHub REST API to get find dependency update PRs for the configured
+# reqositories.
+#
+# This script can simply be run as "./git_hub_links.rb" or "ruby git_hub_links.rb"
+#
+# The GitHub REST API will rate limit you, but not before you would be able to run this script multiple times.
+# If you run into rate limiting issues, you can generate a personal access token and pass it in using the GH_ACCESS_TOKEN env var,
+# "GH_ACCESS_TOKEN=abc123 ./git_hub_links.rb"
+# See https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/ for more info
+class GitHubLinks
+  def self.render
+    new.render
+  end
+
+  def render
+    SlackBot.new.message(text: slack_bot_content)
+  end
+
+  private
+
+  def slack_bot_content
+    repo_content = repos.map do |repo|
+      PullRequest.new(repo)
+    end.map(&:to_s).join("\n")
+
+    [
+      '*Weekly dependency update time is here!*',
+      "\nBelow you will find the content for our weekly dependency update spreadsheet",
+      "```#{repo_content}```"
+    ].join(' ')
+  end
+
+  def repos
+    [js_repos_file, ruby_repos_file].map(&:to_a).flatten.map(&:strip).sort
```

Comment:
This kind-of forces a sort for our projects, but maybe alpha is a good way to go?

Reply:
Alpha is dandy 👍